.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python: 
+  - 3.6

```

Comment:
any reason you chose 3.6 instead of 3.8?

Reply:
in fact the build failure is due to this, can't import the file that uses dataclasses because that is a 3.8+ feature